esx_service/vmdk_ops_test.py

```diff
@@ -722,15 +722,21 @@ def create_default_tenant_and_privileges(self):
 
         if not tenant_uuid:
             logging.debug("create_default_tenant_and_privileges: create DEFAULT tenant")
-            error_info, tenant = auth_api._tenant_create(
+            error_info, auth_mgr = auth_api.get_auth_mgr_object()
+            if error_info:
+                err = error_code.TENANT_CREATE_FAILED.format(auth.DEFAULT_TENANT, error_info.msg)
```

Comment:
That error hit while getting auth manager object will be captured in error_info.msg, and will be included in the error message. The error message will be like "_DEFAULT tenant creates fail with err ...."